test/e2e/forceRender.test.ts

```diff
@@ -38,22 +38,4 @@ describe('forceRender', () => {
       console.log('Zimcheck not installed, skipping test')
     }
   })
-
-  test('Scrape article from bm.wikipedia.org should throw error when using VisualEditor render', async () => {
```

Comment:
Unfortunately, this set of tests should be removed - they caused issues in CI pipeline related to memory corruption. One of the examples of this problem can be found in these logs - https://github.com/openzim/mwoffliner/actions/runs/6854615228/job/18637919124#step:7:1589
I also faced with `terminate called after throwing an instance of 'Xapian::DatabaseLockError'
Aborted (core dumped)` and `Node error process completed with exit code 139`. Probably, that happened because we forced the application to throw an error here without the right clearing of the Redis state and/or dangling process.exit(1) invocation.